Program/stud_fees_data.py

```diff
@@ -0,0 +1,58 @@
+#Program [ 1 ]
+#Create table stud(id, name, dept, paidfee) with 8 records.
+#Create stud.csv file of the stud data (Department will be BBA and BCA).
+#Read csv file using reader object and write data with remaining fee.
+#( Total fee of bba is 13400 and bca is 15400 Rs)
+
+import csv
+
+#Read the CSV file and print data from the file with append()
+#with skipping the header
+
+students = []
+
+#with open("D:\\sqlite\\csv\\stud1.csv", "r") as f:
+f=open("D:\\sqlite\\csv\\stud1.csv", "r")
+    r = csv.reader(f)
+    next(r)
+    for row in r:
+        if all(row):
+            students.append(row)
+print(students)
+f.close()

```

Comment:
why you close the file, it is compulsarry ?

Reply:
> why you close the file, it is compulsarry ?

Yes its compulsory to close the file b'coz i opened the file by using open( )